src/resources/i18n/translations/en.json

```diff
@@ -15,8 +15,8 @@
   "Account": "Account",
   "Active Downloads": "Active Downloads",
   "Actual amount will be higher due to network fees": "The actual amount also includes network fees",
-  "Add current item next in queue when playing a new item": "Add current item next in queue when playing a new episode",
-  "Add Custom RSS Feed": "Add Custom RSS Feed",
+  "Add current item next in queue when playing a new item": "Immediate playback",
```

Comment:
```suggestion
  "Add current item next in queue when playing a new item": "Play newly added playlist items first",
```

Reply:
Sorry, I don't think this really explains the feature either. A "Playlist" is a separate concept than a "Queue" in Podverse.  In theory we could add all items from a Playlist to a Queue, but otherwise they're separate functionality. Have you tried the feature in the app? If you need instructions lmk. It's really hard for me to explain succinctly.